rd_ui/app/login.html

```diff
@@ -44,7 +44,7 @@
   <form role="form" method="post" name="login">
     <div class="form-group">
       <label for="inputEmail">Email</label>
-      <input type="text" class="form-control" id="inputEmail" name="email" value="{{email}}">
+      <input type="email" class="form-control" id="inputEmail" name="email" value="{{email}}">
```

Comment:
While it makes sense, it's a problem as sometimes "email" is used just as a username -- like the default `admin` user. Changing type here will prevent people from logging in...